content/doc/book/blueocean/creating-pipelines.adoc

```diff
@@ -0,0 +1,117 @@
+---
+layout: section
+title: Creating Pipelines in Blue Ocean
+---
+:description:
+:author:
+:email: jenkinsci-docs@googlegroups.com
+:sectanchors:
+:toc:
+:imagesdir: /doc/book/resources
+:hide-uri-scheme:
+
+= Creating Pipelines in Blue Ocean
+
+Blue Ocean makes it easy to create Pipelines in Jenkins.
+Pipelines can be created from existing `Jenkinsfile`s
+or from new `Jenkinsfile`s created with the
+<<pipeline-editor, Blue Ocean Pipeline Editor>>.
+The Pipeline Creation workflow guides users through this process in clear,
+easy to understand steps.
+
+== Starting Pipeline Creation
+
+At the top of the <<dashboard, Blue Ocean Dashboard>>,
+is a "*New Pipeline*" button that launch the Pipeline Creation workflow.
+
+image:blueocean/creating-pipelines/new-pipeline-button.png['Blue Ocean New Pipeline Button']
+
+On new a Jenkins instance, where there are no jobs or Pipelines
+and the Dashboard is empty, Blue Ocean will also display a message box offering
+to "Create a New Pipeline".
+
+image:blueocean/intro/new-pipeline-box.png['Blue Ocean Create Pipeline Message Box']
+
+== Creating a Pipeline for a Git Repository
+
+To create a Pipeline from a Git repository,
+start by selecting "Git" as the Source Control system.
+
+image:blueocean/creating-pipelines/select-git.png['Creating Pipeline from a Git Repository']
+
+Then enter the URL for the Git Repository and optionally select which credentials
+to use.  If the dropdown down does not show the desired credentials,
+they can be added using the "Add" button.
+
+When done, click "Create Pipeline".
+Blue ocean will look at all branches for the selected repository,
+and will start a Pipeline run for each branch containing a `Jenkinsfile`.
+
+image:blueocean/creating-pipelines/git-create-pipeline.png['Providing Git Repository Details']
+
+== Creating Pipelines for GitHub Repositories
+
+To create a Pipeline from a GitHub,
+start by selecting "GitHub" as the Source Control system.
+
+image:blueocean/creating-pipelines/select-github.png['Creating Pipeline for Github Repositories']
+
+=== Provide a GitHub Access Token
+
+If this is the first time Pipeline Creation has been run,
```

Comment:
"has been run by the currently logged in user" or something along those lines. From my recollection the personal access token is associated with the logged in user, rather than the system, so each user creating a Pipeline would need to add their own token.